### DISCUSSION TITLE
integrate cesium-native into rocky

### DISCUSSION THREAD

**Discussion by xiaoxin0214** [pelicanmapping/rocky#21] [Q&A / Support]
Is it possible to integrate cesium-native into rocky like what osgearth project did? https://cesium.com/blog/2023/10/05/osgearth-adds-support-for-cesium-native/

**Comment by gwaldron** [comment]
It is definitely possible, and someday we might do so.
In the meantime you can check out the [vsgCS](https://github.com/timoore/vsgCs) project which integrates VSG with Cesium Native.
